source/guides/models/repositories.md

```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
   # => nil
 ```
 
+## Saving more than one record
+
+For saving bunch of records you need to create `#create_many` command:
+
+```ruby
+class BookRepository
+  def create_many(list)
+    command(:create, books, result: :many).call(list)
+  end
+end
+```
+
+Now you can use `#create_many` a bulk creation method:
+
+```ruby
+repository = BookRepository.new
+
+repository.create_many([{ title: "Hanami" }, { title: "Ruby"}])
```

Comment:
Since it's just one command, should we do `BookRepository.new.create_many`, instead of defining a local variable?